src/Faithlife.WebRequests/WebServiceRequestBase.cs

```diff
@@ -145,52 +145,27 @@ protected WebServiceRequestBase(Uri uri)
 		/// <summary>
 		/// Gets the response.
 		/// </summary>
-		/// <returns>Makes a web request, blocking until the request has returned a response.</returns>
-		public TResponse GetResponse()
-		{
-			return GetResponse(WorkState.None);
-		}
-
-		/// <summary>
-		/// Gets the response.
-		/// </summary>
-		/// <param name="workState">The work state.</param>
+		/// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
 		/// <returns>Makes a web request, blocking until the request has returned a response. Returns default(TResponse) if cancelled.</returns>
-		public TResponse GetResponse(IWorkState workState)
+		public TResponse GetResponse(CancellationToken cancellationToken = default(CancellationToken))
```

Comment:
Supporting cancellation on sync methods is highly unusual. If we must keep the sync method, we should drop the cancellation token; clients that need both that block on the async method themselves.